### ISSUE TITLE
Flowchart for plant_flow

### ISSUE THREAD

**Issue by pgebraad** [FUSED-Wind/fusedwind#69]
I have been mainly working with fusedwind.plant_flow so far. I am working to implement my FLORIS wake model, and then later connect it to rotor characteristics (ROTORSE, CC-BLADED). Although the tutorials are helpful to get started, still I am experiencing quite a steep learning curve to understand why some parts connect to others parts. What makes this more difficult too, is that there is a lot of backwards compatibility which makes it harder to separate the main part of the code from old stuff. Therefore, adding on the website a general flowchart with the main components of the fusedwind.plant_flow would be helpful maybe.

Just a simple example for the GenericWindFarm component:
![flowchart](https://cloud.githubusercontent.com/assets/9433870/6628681/2e0c5fee-c8cd-11e4-8a58-2acef02d62e9.png)

Pieter Gebraad (NREL)

**Comment by piredtu** [comment]
Yes, I agree. There is a lot of cleaning up to do. Until now I was pretty much alone on this part of the code, so I haven't been too motivated to do it. The priority was to make it work and not break backward compatibility. But now that you are going to use it, I'm going to prioritise it. 
I plan to do it while adapting the code to openmdao 0.12+
